crates/bevy_ecs/src/schedule/state.rs

```diff
@@ -1,4 +1,5 @@
-use crate::{
+
+use crate::{use crate::{
```

Comment:
```suggestion
use crate::{
```

This should help fix the CI issues.